src/ui/components/SearchInput/style.scss

```diff
@@ -20,19 +20,16 @@ $transition-duration: 250ms;
 
     vertical-align: top;
   }
+}
 
-  .SearchInput-animation-icon {
-    @include start(0);
+.SearchInput-animation-icon {
+  @include start(0);
 
-    opacity: 0;
-    position: absolute;
-    transition: opacity 1ms $transition-duration, transform $transition-duration;
-  }
+  opacity: 0;
+  position: absolute;
 }
 
 .SearchInput--text {
-  @include text-align-start();
```

Comment:
This was just moving the label to the left which is hidden in this state so there was no point (and it might have contributed to messing up position calculations in `setIconPosition`.